pyproject.toml

```diff
@@ -12,5 +12,6 @@ exclude = '''
   | buck-out
   | build
   | dist
+  | cdk/venv
```

Comment:
This is the black config file - this is just telling it to ignore the virtual environment folder in cdk (if you use one)